backend/data_preparation/dumper/fire_dumper.py

```diff
@@ -0,0 +1,438 @@
+"""
+@author: Scarlett Zhang
+This file has 2 classes:
+InvalidRecordError
+FireDumper
+"""
+from typing import List, Dict, Tuple
+import logging
+import rootpath
+from backend.data_preparation.dumper.dumperbase import DumperBase
+from backend.data_preparation.connection import Connection
```

Comment:
the newest master has relocated the connection class. have you tested with the new master?